src/ol/PluggableMap.js

```diff
@@ -538,8 +538,7 @@ class PluggableMap extends BaseObject {
    * callback with each intersecting feature. Layers included in the detection can
    * be configured through the `layerFilter` option in `opt_options`.
    * @param {import("./pixel.js").Pixel} pixel Pixel.
-   * @param {function(this: S, import("./Feature.js").FeatureLike,
```

Comment:
The description should be updated to include the third geometry parameter.
The `FeatueCallback<T>` type is not available for the apidocs, becomes a dead link.

Reply:
I intentionally did not document that third parameter, because I am not sure it is relevant for the API. The reason why I added it was to be able to use the Modify interaction with hit detection and geometry collections after #11781, where this third parameter is used internally.

I did update the type to avoid the dead link, so the documentation should now be the same as before this pull request.